.github/pull_request_template.md

```diff
@@ -35,9 +35,20 @@ Please mark with X for applicable areas.
 | Other                      | [ ] |
 
 
+**Testing done**:
+
+<!--
+
+Please describe how are the changes tested? You can add supporting information, E.g. screenshots, logs etc.
+
+-->
+
+
+**Miscellaneous**
+
 Please answer the following questions with yes/no.
 
 1. Does this change contain code from or inspired by another project?
     -   Did you notify the maintainers and provide attribution?
 
-1. Is this a breaking change?
+2. Is this a breaking change?
```

Comment:
Not needed. Markdown will correctly render this as 2.

Reply:
Didn't mean to change it. It's updated by my editor. Will revert